locales/ja_JP/LC_MESSAGES/duckduckgo.po

```diff
@@ -195,7 +195,7 @@ msgstr "DuckDuckGo を追加する"
 
 msgid ""
 "Add DuckDuckGo Privacy Essentials to your browser for free with one download:"
-msgstr ""
+msgstr "DuckDuckGoプライバシーエッセンシャルは1度のダウンロードで、ブラウザに無料で追加できます："
```

Comment:
Looks like Privacy Essentials was translated here. @tagawa might be able to help us fix this translation?

Reply:
I assumed it was ok since that's phonetic, so when said aloud it sounds like "Privacy Essentials".  But it looks like we didn't do that elsewhere: https://github.com/duckduckgo/duckduckgo-locales/blob/3c2e70ac3bd61ec619f0b0a0a03ac35f2461f5da/locales/ja_JP/LC_MESSAGES/duckduckgo.po#L1342

Curious what @tagawa's perspective is